apps/synapse/1.119.0/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       - TZ=Asia/Shanghai
       - UID=1000
       - GID=1000
-    image: matrixdotorg/synapse:v1.119.0
+    image: matrixdotorg/synapse:v1.120.0
     logging:
       options:
         max-size: "10m"
```

Comment:
I have no issue with this code snippet. It appears to be properly structured and formatted based on the information provided. It sets up a Docker container using Synapse from Matrix.org with specific configurations such as timestamp, user ID (UID), group ID (GID), and logging parameters. There does not seem to be any known issue or optimization that can be suggested without more context.